py/torch_tensorrt/fx/converters/operator.py

```diff
@@ -0,0 +1,1256 @@
+    )
+    set_layer_name(mean_trt_layer, target, f"{name}_mean")
+    # Variance + eps
+    eps_tensor = network.add_constant(
+        (1,) * len(input_val.shape),
+        trt.Weights(np.ascontiguousarray([eps], dtype=np.float32)),
+    )
+    eps_tensor.name = f"{name}_eps"
+    add_trt = add_binary_elementwise_layer(
+        network,
+        mean_trt_layer.get_output(0),
+        eps_tensor.get_output(0),
+        trt.ElementWiseOperation.SUM,
+        target,
+        f"{name}_add",
+    )
+    # SQRT((Var + eps))
+    sqrt_trt = add_unary_layer(
+        network, add_trt, trt.UnaryOperation.SQRT, target, f"{name}_sqrt"
+    )
+    # (x - E[x]) / sqrt((var + eps))
+    div_trt = add_binary_elementwise_layer(
+        network,
+        sub_trt,
+        sqrt_trt,
+        trt.ElementWiseOperation.DIV,
+        target,
+        f"{name}_div_trt",
+    )
+
+    assert gamma is not None
+    gamma_tensor = network.add_constant(gamma.shape, trt.Weights(np.ascontiguousarray(gamma)))  # type: ignore[attr-defined]
+    gamma_tensor.name = f"{name}_gamma"
+    assert beta is not None
+    beta_tensor = network.add_constant(gamma.shape, trt.Weights(np.ascontiguousarray(beta)))  # type: ignore[attr-defined]
+    beta_tensor.name = f"{name}_beta"
+    # y * gamma + beta
+    scale_layer = add_binary_elementwise_layer(
+        network,
+        div_trt,
+        gamma_tensor.get_output(0),
+        trt.ElementWiseOperation.PROD,
+        target,
+        f"{name}_scale",
+    )
+    return add_binary_elementwise_layer(
+        network,
+        scale_layer,
+        beta_tensor.get_output(0),
+        trt.ElementWiseOperation.SUM,
+        target,
+        name,
+    )
+
+
+def add_add(network, target, kwargs, name):
+    return add_binary_elementwise_layer(
+        network,
+        kwargs["input"],
+        kwargs["other"],
+        trt.ElementWiseOperation.SUM,
+        target,
+        name,
+    )
+
+
+def add_matmul(network, target, kwargs, name):
+    input_val = get_trt_tensor(network, kwargs["input"], f"{name}_input")
+    other_val = get_trt_tensor(network, kwargs["other"], f"{name}_other")
+
+    for i in [input_val, other_val]:
+        if not isinstance(i, TRTTensor):
+            raise RuntimeError(
+                f"matmul received input {i} that is not part of the TensorRT region!"
+            )
+
+    input_matrix_op = other_matrix_op = trt.MatrixOperation.NONE
+    preset_diff = 0
+
+    if len(input_val.shape) == 1:
+        preset_diff -= 1
+        input_matrix_op = trt.MatrixOperation.VECTOR
+
+    if len(other_val.shape) == 1:
+        preset_diff += 1
+        other_matrix_op = trt.MatrixOperation.VECTOR
+
+    input_val, other_val = broadcast(
+        network, input_val, other_val, f"{name}_input", f"{name}_other", preset_diff
+    )
+    layer = network.add_matrix_multiply(
+        input_val, input_matrix_op, other_val, other_matrix_op
+    )
+    set_layer_name(layer, target, name)
```

Comment:
Some layer name changes use this function and others directly modify `layer.get_output(0).name`. Consider standardizing the usage across functions.

Reply:
That makes sense.

Reply:
That makes sense. Just that in the case of `set_layer_name` the name is changed directly for the layer, while appending `acc_ops.target.__name__` to the layer name with the name passed in the above function. Whereas in case of the other we change the name of `layer.get_output(0)` with only the `target.__name__`. @frank-wei any thoughts on this to integrate the two?